db/codeUpdated/mainnet/legacyStats.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO stat_types (id, asset_id, op_type, created_block_id, updated_block_id, created_at, updated_at ) VALUES ('20220824/Count', '20220824', 'Count', '1', '1', NOW(), NOW());
```

Comment:
this should probably be removed, but for now this is the patch to fix current versions